tutorials/rnn/translate/seq2seq_model.py

```diff
@@ -114,12 +114,14 @@ def sampled_loss(labels, inputs):
       softmax_loss_function = sampled_loss
 
     # Create the internal multi-layer cell for our RNN.
-    single_cell = tf.nn.rnn_cell.GRUCell(size)
+    def single_cell():
+      return tf.nn.rnn_cell.GRUCell(size)
```

Comment:
I noticed this code is out of date (`tf.nn.rnn_cell` rather than `tf.contrib.rnn`), and it appears the source of this is this commit: https://github.com/tensorflow/models/commit/c902a86785052291083682c4b833747e7ea52597#diff-4826da5b71f89d05b5b53ded2bd2e138.

Will add another PR to fix some of these changes.